core-bundle/src/Resources/contao/templates/analytics/analytics_google.html5

```diff
@@ -13,4 +13,4 @@ if ('UA-XXXXX-X' != $GoogleAnalyticsId && !BE_USER_LOGGED_IN && !$this->hasAuthe
   ga('send', 'pageview');
 </script>
 
-<?php endif; ?>
+<?php endif; ?>
```

Comment:
This change should probably reverted.

Reply:
well either these templates are wrong, or our .editorconfig is wrong, because it says there should be no final newline in .html5 files?

Reply:
Then the editorconfig is wrong IMO. html5 files should have a final newline AFAIK.

Reply:
see https://github.com/contao/contao/pull/3877

Reply:
fixed in ad48c1cf7f5e6071f4f59063aa7e498dd592e07d. see https://github.com/contao/contao/pull/3877 for the `.editorconfig` changes.